app/scripts/views/settings/communication_preferences.js

```diff
@@ -93,35 +94,35 @@ define(function (require, exports, module) {
       return this.setOptInStatus(NEWSLETTER_ID, ! emailPrefs.isOptedIn(NEWSLETTER_ID));
     },
 
-    setOptInStatus (newsletterId, isOptedIn) {
-      let eventPrefix;
-      let method;
+    setOptInStatus (newsletterId, isOptingIn) {
```

Comment:
I changed the naming here because above it is actually the opposite of the current opt in status: ` return this.setOptInStatus(NEWSLETTER_ID, ! emailPrefs.isOptedIn(NEWSLETTER_ID));`